chain/gen/gen.go

```diff
@@ -121,7 +121,7 @@ var DefaultRemainderAccountActor = genesis.Actor{
 }
 
 func NewGeneratorWithSectors(numSectors int) (*ChainGen, error) {
-	saminer.SupportedProofTypes = map[abi.RegisteredSealProof]struct{}{
+	miner0.SupportedProofTypes = map[abi.RegisteredSealProof]struct{}{
```

Comment:
Need to handle this for both versions, probably easiest done with a setter in the abstraction layer which would just set both

Reply:
Good point.

Reply:
It would be kind nice if these were actually specified by the runtime. But I guess that's more of a future consideration (too much work now).